README.md

```diff
@@ -62,52 +77,53 @@ public class Student {
     @Id
     private int id;
 
-		@Column(name = "student_name")
+    @Column(name = "student_name")
     private String name;
+    
 
-		private int age;
-
-		// getters and setters
+    // getters and setters
 }
 ```
 
 We can use several values such as `name`, `length`, `unique`, `nullable` which
 can modify the column name or set database constraints. For example, the `name`
-property’s column name in the database will be set to “student_name” instead of
-“name”.
+property’s column name in the database will be set to "student_name" instead of
+the default "name".
 
 You can check out the different options and their usage
 [here](https://www.objectdb.com/api/java/jpa/Column) and
 [here](https://docs.oracle.com/javaee/7/api/javax/persistence/Column.html).
 
 ## Handling Different Data Types
 
-We might have to use non-primitive data types such as `Date`, `Enums` in our
+We might have to use non-primitive data types such as `Date` or `Enum` in our
 Java program. We have to specify how we want them to be mapped to the database
-because relying on the defaults may not be what you want.
+because relying on the defaults may not provide the desired functionality.
 
 ### Dates
 
-We will remove the `age` property and add a `dob` (Date of Birth) property to
+Let's add a `dob` (Date of Birth) property to
 our `Student` class. The `@Temporal` annotation is used to define how the
 information will be stored in the database. We are using the `Date` class here
 to demonstrate the `@Temporal` annotation. For production apps, you would use
 the `LocalDate` class instead of the `Date` class.
```

Comment:
If LocalDate is preferred for production apps, should we use that here instead, too?

Reply:
I agree with this statement.

Reply:
updated to use LocalDate